tests/integration/update/test_rename_executor.py

```diff
@@ -0,0 +1,92 @@
+import os
+
+from jina import Client, Document, DocumentArray, Flow
+
+from jcloud.flow import CloudFlow
+
+from jcloud.helper import get_dict_list_key_path, remove_prefix
+
+flows_dir = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'flows')
+flow_file = 'base_flow.yml'
+rename_executor_flow_file = 'custom_name_executor.yml'
+expsoed_executor_flow_file = 'expose_executor.yml'
+rename_expsoed_executor_flow_file = 'custom_name_exposed_executor.yml'
+protocol = 'http'
+
+def test_rename_executor():
+    with CloudFlow(path=os.path.join(flows_dir, flow_file)) as flow:
+
+        assert flow.endpoints != {}
+        assert 'gateway' in flow.endpoints
+        gateway = flow.endpoints['gateway']
+        assert gateway.startswith(f'{protocol}s://')
+
+        da = Client(host=gateway).post(
+            on='/',
+            inputs=DocumentArray(Document(text=f'text-{i}') for i in range(50)),
+        )
+        assert len(da.texts) == 50
+
+        flow.path = os.path.join(flows_dir, rename_executor_flow_file)
+        flow._loop.run_until_complete(flow.update())
+
+        assert flow.endpoints != {}
+        assert 'gateway' in flow.endpoints
+        gateway = flow.endpoints['gateway']
+        assert gateway.startswith(f'{protocol}s://')
+
+        status = flow._loop.run_until_complete(flow.status)
+
+        assert get_dict_list_key_path(status, ['spec', 'executors', 0, 'name']) == 'newsentencizer'
```

Comment:
Don't quite follow what's being done here, that `status` should have a list of keys?

Reply:
Also black linter broke, please fix.

Reply:
instead of adding too many assertions, it simply asserts all the keys in nested manner.
it is equivalent to `status['spec']['executors'][0]['name]` with exception handling at each level.

Reply:
oh ok I guess this will work